docs/versioned_docs/version-v2.4.0/02-Features/endpoint-validation.md

```diff
@@ -31,14 +31,14 @@ There are 2 attributes available to secure operations:
 These attributes will only allow an operation to be called, if the configured secret key value is found in the HTTP header or the Query String with the configured key name.
 
 #### The EventGridAuthorization attribute
-This attribute has the following public constructor.  This constructor sets the header or querystring name and the hard coded secret value.
+This attribute has the following public constructor.  This constructor sets the header or query string name and the hard coded secret value.
```

Comment:
Hmm, I'm not sure if it is querystring or query string.

Reply:
I searched it online to find it 😄 
https://en.wikipedia.org/wiki/Query_string